File: app/src/main/java/com/alesno/testtaskispring/model/objectbox/entity/ExpertObj.java
package com.alesno.testtaskispring.model.objectbox.entity;

import io.objectbox.annotation.Entity;
import io.objectbox.annotation.Id;
import io.objectbox.annotation.NameInDb;
import io.objectbox.relation.ToOne;

@Entity
public class ExpertObj {

    @Id
    public long id;

    public String avatar;

    @NameInDb("first_name")
    public String firstName;

    @NameInDb("id_response")
    public int idResponse;

    @NameInDb("second_name")
    public String secondName;

    public String speciality;

    public ToOne<VideoObj> video;

}

File: app/src/main/java/com/alesno/testtaskispring/model/objectbox/entities/ExpertObj.kt
package com.alesno.testtaskispring.model.objectbox.entities

import io.objectbox.annotation.Id
import io.objectbox.annotation.NameInDb
import io.objectbox.relation.ToOne

data class ExpertObj(

    @Id var id: Long = 0,

    var avatar: String? = null,

    @NameInDb("first_name")
    var firstName: String? = null,

    @NameInDb("id_response")
    var idResponse: Int = 0,

    @NameInDb("second_name")
    var secondName: String? = null,

    var speciality: String? = null
){

    lateinit var video: ToOne<VideoObj>
}